test/key_test.py

```diff
@@ -24,19 +24,24 @@ def test_import(self):
         "does it import"
         import pygame.key
 
-    def todo_test_get_focused(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.key.get_focused:
-
-        # pygame.key.get_focused(): return bool
-        # true if the display is receiving keyboard input from the system
-        #
-        # This is true when the display window has keyboard focus from the
-        # system. If the display needs to ensure it does not lose keyboard
-        # focus, it can use pygame.event.set_grab() to grab all input.
-        #
-
-        self.fail()
+    def test_get_focused(self):
+        focused = pygame.key.get_focused()
+        if not SDL1:
+            self.assertFalse(focused) #No window to focus
```

Comment:
Does this mean that under SDL1 `key.get_focused()` returns True, even before you call `display.set_mode()` to create a window? That seems a bit screwy and maybe something we could fix in the C code.

Reply:
Yes, it appears that key.get_focused() returns true with SDL1 without a window to focus, at least on the build test machine. I do think it is odd, but I assumed the function was working as currently written.